dataland-external-storage/src/main/resources/application.properties

```diff
@@ -20,6 +20,4 @@ spring.rabbitmq.password=${RABBITMQ_PASS}
 spring.main.allow-bean-definition-overriding=true
 dataland.eurodatclient.base-url=${INTERNAL_EURODATCLIENT_URL:http://eurodat-client:8080}
 dataland.eurodatclient.initialize-safe-deposit-box=true
-dataland.eurodatclient.app-name=minaboApp
-dataland.eurodatclient.max-retries-connecting=8
-dataland.eurodatclient.milliseconds-between-retries=15000
\ No newline at end of file
+dataland.eurodatclient.app-name=minaboApp
```

Comment:
Fände es schon cleaner, wenn das app props bleiben.
Jetzt wo die retry-Logik ausgelagert wurde nach `EurodatDataStoreUtils`, könnten wir das ja wieder so machen, weil das detekt-Problem mit den zu vielen Params nicht mehr im Weg steht.

Reply:
Die lassen sich aber nicht in das Object "EurodatDataStoreUtils" autowiren, deshalb sind die da hardgecoded.

Reply:
Plus die genauen Werte ändern sich auch nicht je nachdem auf welcher Umgebung deployed wird, daher greift ja der besondere Vorteil der application_properties incht, dass man die Werte umgebungsspezifisch unterschiedlich setzen kann.

Reply:
Ok ich gucks mir nochmal an und schließe das hier entweder, oder komme mit Gegenargumenten.